packages/ui-extensions/src/surfaces/admin/components/DatePicker/DatePicker.doc.ts

```diff
@@ -0,0 +1,43 @@
+import {ReferenceEntityTemplateSchema} from '@shopify/generate-docs';
+
+const data: ReferenceEntityTemplateSchema = {
+  name: 'DatePicker',
+  description:
+    'Date pickers let merchants choose dates from a visual calendar that’s consistently applied wherever dates need to be selected across Shopify.    ',
+  requires: '',
+  thumbnail: 'datepicker-thumbnail.png',
+  isVisualComponent: true,
+  type: '',
+  definitions: [
+    {
+      title: 'DatePickerProps',
+      description: '',
+      type: 'DatePickerProps',
+    },
+  ],
+  category: 'Components',
+  subCategory: 'Forms',
+  defaultExample: {
+    image: 'datepicker-default.png',
+    codeblock: {
+      title:
+        'Use when merchants need to select a single day, multiple dates, or a range of dates',
```

Comment:
I think this needs to be shorter to look good. For example look at Badge:

<img width="614" alt="Screenshot 2023-11-07 at 9 41 37 AM" src="https://github.com/Shopify/ui-extensions/assets/5873610/0f4e431a-1e84-47c0-82f9-e123201415d8">